Microsoft.Identity.Web.Aspnet/MicrosoftIdentityWebApiAuthorizationBuilder.cs

```diff
@@ -17,10 +19,12 @@ public static void ProtectWebApiWithMicrosoftIdentity(this IAppBuilder app, JwtB
                 new WindowsAzureActiveDirectoryBearerAuthenticationOptions
                 {
                     Tenant = webApiConfig.TenantId,
+
                     TokenValidationParameters = new TokenValidationParameters
                     {
                         SaveSigninToken = true,
-                        ValidAudiences = webApiConfig.ValidAudiences
+                        ValidAudiences = webApiConfig.ValidAudiences,
+                        ValidIssuers = CommonUtil.BuildValidIssuers(webApiConfig)
```

Comment:
Did you consider using ValidateIssuer instead ? (and set it to the AadIssuerValidator)? 
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/039d09b7bc4c1671e1c4093d5a62d307c2f96935/src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs#L155-L156

Same for the other PR, btw

Reply:
@jmprieur Thank you, I will check it

Reply:
thats what he will do next

Reply:
@jmprieur I'm trying to use the code you've refenced, but cannot instantiate the validator class because it's constructor is internal. Do I miss something?

Reply:
@jmprieur done proposed change